packages/components/select/select.component.ts

```diff
@@ -444,7 +444,7 @@ export class KbqSelect
      * Minimum width of the panel.
      * If minWidth is larger than window width, it will be ignored.
      */
-    @Input({ transform: numberAttribute }) panelMinWidth: number;
+    @Input({ transform: numberAttribute }) panelMinWidth: number = 200;
```

Comment:
давай сделаем по аналогии с `panelWidth`, чтобы можно было глобально настраивать:
```suggestion
    @Input({ transform: numberAttribute }) panelMinWidth: number = this.defaultOptions?.panelWidth || 200;
```